### PR TITLE
Test suite

### PR DESCRIPTION
**Pull request by mssalvatore**
Add unit tests for python version of cvescan. Correct a few bugs detected while developing the test suite.

### CODE REVIEW

File: tests/assets/lsb-release-not-ubuntu [hunk 1]
```diff
@@ -0,0 +1,4 @@
+DISTRIB_ID=not-ubuntu
```
Comment (markmorlino) [inline tests/assets/lsb-release-not-ubuntu:1]: hehe

File: cvescan/cvescanner.py [hunk 2]
```diff
@@ -146,50 +146,60 @@ def _count_packages_in_manifest_file(manifest_file):
...
 def _analyze_single_cve_results(cve_list_all_filtered, cve_list_fixable_filtered, cve):
     if cve in cve_list_fixable_filtered:
-        return ("%s patch available to install" % cve, const.PATCH_AVAILABLE_RETURN_CODE)
+        return ("A patch is available to fix %s." % cve, const.PATCH_AVAILABLE_RETURN_CODE)
 
     if cve in cve_list_all_filtered:
-        return ("%s patch not available" % cve, const.SYSTEM_VULNERABLE_RETURN_CODE)
+        return ("%s affects this system, but no patch is available." % cve, const.SYSTEM_VULNERABLE_RETURN_CODE)
+
+    return ("This system is not affected by %s." % cve, const.SUCCESS_RETURN_CODE)
```
Comment (markmorlino) [inline cvescan/cvescanner.py:182]: This is strong language. I'm not sure I have enough confidence in all of our UCT data and OVAL generation to support such a definite assertion.

Comment (mssalvatore) [reply cvescan/cvescanner.py:182]: How about, "This system is not known to be affected by %s."?

Comment (markmorlino) [reply cvescan/cvescanner.py:182]: I like it better

**Pull request merged**